gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,8 +1,7 @@
-# Copyright (c) 2020-2023 Tesla (Yinsen) Zhang.
-# Use of this source code is governed by the MIT license that can be found in the LICENSE.md file.
```

Comment:
Shall not delete this

Reply:
I can add it back, while this file is a generated file and generally we can place license header for such file - I'm curious why you think it's necessary or just preference.

Reply:
The license is also generated

Reply:
Added back.